lib/websocket-client-simple/version.rb

```diff
@@ -1,7 +1,7 @@
 module WebSocket
   module Client
     module Simple
-      VERSION = "0.3.0"
+      VERSION = "0.3.3"
```

Comment:
Don't edit the version, please. The version is changed at release by the maintainer.